<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1137</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(234, 241, 237);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="colapsed_bar_widget" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget {
	background-color: rgb(85, 170, 127);
}

QLabel {
	color: white;
}

QPushButton {
	height: 30px;
	border: none;
}

QPushButton::checked {
	background-color: rgb(220, 255, 236);
	color: rgb(85, 170, 127);
	font-weight: bold;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../icon_resources.qrc">:/icons/home.png</pixmap>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>15</number>
         </property>
         <item>
          <widget class="QPushButton" name="col_dashboard">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../icon_resources.qrc">
             <normaloff>:/icons/dashboard_white.png</normaloff>:/icons/dashboard_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="col_gestion">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../icon_resources.qrc">
             <normaloff>:/icons/users-alt.png</normaloff>:/icons/users-alt.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="col_listado">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../icon_resources.qrc">
             <normaloff>:/icons/list-check.png</normaloff>:/icons/list-check.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>419</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../icon_resources.qrc">
           <normaloff>:/icons/log_out_white.png</normaloff>:/icons/log_out_white.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="expanded_bar_widget" native="true">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {
	background-color: rgb(85, 170, 127);
}

QLabel {
	color: white;
}

QPushButton {
	color: white;
	text-align: left;
	height: 30px;
	border: none;
	padding-left:10px;
	border-top-left-radius: 10px;
	border-bottom-left-radius: 10px;
}

QPushButton::checked {
	background-color: rgb(220, 255, 236);
	color: rgb(85, 170, 127);
	font-weight: bold;
}
</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../icon_resources.qrc">:/icons/home.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Home</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>17</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>15</number>
         </property>
         <item>
          <widget class="QPushButton" name="exp_dashboard">
           <property name="text">
            <string>Dashboard</string>
           </property>
           <property name="icon">
            <iconset resource="../icon_resources.qrc">
             <normaloff>:/icons/dashboard_white.png</normaloff>:/icons/dashboard_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exp_gestion">
           <property name="text">
            <string>Gestion</string>
           </property>
           <property name="icon">
            <iconset resource="../icon_resources.qrc">
             <normaloff>:/icons/users-alt.png</normaloff>:/icons/users-alt.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exp_listado">
           <property name="text">
            <string>Listado</string>
           </property>
           <property name="icon">
            <iconset resource="../icon_resources.qrc">
             <normaloff>:/icons/list-check.png</normaloff>:/icons/list-check.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>418</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_8">
         <property name="text">
          <string>Sign Out</string>
         </property>
         <property name="icon">
          <iconset resource="../icon_resources.qrc">
           <normaloff>:/icons/log_out_white.png</normaloff>:/icons/log_out_white.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="main_screen_widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border: none;
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="menu">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../icon_resources.qrc">
                <normaloff>:/icons/menu.png</normaloff>:/icons/menu.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>538</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_10">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../icon_resources.qrc">
                <normaloff>:/icons/image.png</normaloff>:/icons/image.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="dashboard"/>
         <widget class="QWidget" name="listado"/>
         <widget class="QWidget" name="opciones"/>
         <widget class="QWidget" name="gestion"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../icon_resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>menu</sender>
   <signal>toggled(bool)</signal>
   <receiver>colapsed_bar_widget</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menu</sender>
   <signal>toggled(bool)</signal>
   <receiver>expanded_bar_widget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>col_listado</sender>
   <signal>toggled(bool)</signal>
   <receiver>exp_listado</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>col_gestion</sender>
   <signal>toggled(bool)</signal>
   <receiver>exp_gestion</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>col_dashboard</sender>
   <signal>toggled(bool)</signal>
   <receiver>exp_dashboard</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exp_dashboard</sender>
   <signal>toggled(bool)</signal>
   <receiver>col_dashboard</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>36</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exp_gestion</sender>
   <signal>toggled(bool)</signal>
   <receiver>col_gestion</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>37</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exp_listado</sender>
   <signal>toggled(bool)</signal>
   <receiver>col_listado</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>36</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>657</y>
    </hint>
    <hint type="destinationlabel">
     <x>37</x>
     <y>748</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>650</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>740</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
